db/models.py

```diff
@@ -31,22 +33,81 @@ class CinemaHall(models.Model):
     rows = models.IntegerField()
     seats_in_row = models.IntegerField()
 
-    @property
-    def capacity(self) -> int:
-        return self.rows * self.seats_in_row
-
     def __str__(self) -> str:
         return self.name
 
 
 class MovieSession(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
-    )
-    movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
-    )
+    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
```

Comment:
Code Style issue: It's recommended to use `get_user_model()` instead of `settings.AUTH_USER_MODEL` as a best practice. This way, if the user model changes in the future, you won't need to change your code.